.circleci/workflows.yml

```diff
@@ -822,7 +822,7 @@ commands:
                       name: Run tests using browser "<<parameters.browser>>"
                       working_directory: /tmp/<<parameters.repo>>/<<parameters.folder>>
                       command: |
-                        <<parameters.command>> -- --browser <<parameters.browser>>
+                        <<parameters.command>> --browser <<parameters.browser>> --record false
```

Comment:
was recording disabled to test this? or are you intentionally disabling? 

Also not in this scope per-say, but do you know whats different between the `test-binary-against-rwa` and `test-binary-against-repo` jobs? `test-binary-against-rwa` seems like a copy paste?

Reply:
Recording was never enabled, but i needed to override the project ID and record key to send telemetry. I disabled recording to silence a warning thrown when record is not enabled.

I don't know the different between test-binary-against-repo, I didn't notice they were the same.

Reply:
ohh I see, passing `CYPRESS_PROJECT_ID` prob caused that error

Reply:
Do we no longer need the double `--` since the command is executed with `yarn`?

Reply:
Yeah, yarn kept yelling at me about it.